tests/main/xdg-runtime-dir/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Ensure that XDG_RUNTIME_DIR is created
+
+details: |
+    Ensure that the XDG_RUNTIME_DIR is created when a snap is launched.
+
+restore: |
+    snap remove --purge test-snapd-xdg-dir || true
```

Comment:
The common test cleanup should remove this for you.

Reply:
Done.